### DISCUSSION TITLE
2.1 引言

### DISCUSSION THREAD

**Discussion by giscus** [csfive/composing-programs-zh#106] [Announcements]
# 2.1 引言

SICP Python 中文翻译

https://composingprograms.netlify.app/2/1

**Comment by CharlieCRX** [comment]
😉这里贴一下我的博客，希望能帮到更多的正在攻读这本书的小伙伴：[CS61A阅读指南](https://www.cnblogs.com/shangshankandashu/articles/18046806)

**Reply by wangding** [reply]
太贴心了
